### DISCUSSION TITLE
BA.2 Variant Update?

### DISCUSSION THREAD

**Discussion by zabkillkitten** [microCOVID/microCOVID#1413] [General]
Hello! I was wondering when microCOVID may be updated with additional data regarding the more transmissible BA.2 variant that has been on the rise?

**Comment by brainwane** [comment]
Also requested in https://github.com/microCOVID/microCOVID/issues/1433 .

**Comment by sameerjain0123** [comment]
Hi other folks who seem to care about this - how are you approaching COVID risk management in the meantime? 

I am pretty much following the same behavioral precautions as I did previously when risk was high (n95 or p100 indoors), and just making exceptions for the things that really matter to me and mitigating them the best I can (only attending partner dance events that require high-filtration masks, asking dates to test if we are going to be spending significant time indoors / doing activities that are high-risk).
